packages/waku/src/lib/builder/build.ts

```diff
@@ -686,11 +686,18 @@ export async function build(options: {
   );
 
   const distEntries = await import(filePathToFileURL(distEntriesFile));
+
+  // TODO: proper time measuring and logging
+  let timestamp = Date.now();
+  console.log('Generating build config');
```

Comment:
When the static builds take longer its nice to have at least some feedback. And there is the "TODO" but it felt out of scope for this PR. Should I address this? Either remove the logs or pull in a logging library?

Reply:
Yeah, it might be useful in some cases.
But, we want to reduce logs by default.
However, not a big fan of adding a library at this point.
If you plan a follow-up PR, we can leave it. Otherwise, maybe remove it for now.

Reply:
If we don't add a logging library, there is no follow up PR. I will just leave a `TODO`.